js/src/personal/riyuan/src/css/nav.module.css

```diff
@@ -0,0 +1,350 @@
+:root {
+
+    --body-color: #202224;
+    --shell-color: #171717;
+    --primary-color: #3a3b3c;
+    --primary-color-light: #3a3b3c;
+    --toggle-color: #fff;
+    --text-color: #ccc;
+    --bg-black-900: #151515;
+    --bg-black-100: #222222;
+    --bg-black-50: #393939;
+    --text-black-900: #ffffff;
+    --text-black-700: #e9e9e9;
+}
+
+
+/* 深色主题变量 */
+.dark {
+    --body-color: #E4E9F7;
+    --shell-color: #FFF;
+    --primary-color: #695CFE;
+    --primary-color-light: #F6F5FF;
+    --toggle-color: #DDD;
+    --text-color: #707070;
+    --bg-black-900: #f2f2fc;
+    --bg-black-100: #fdf9ff;
+    --bg-black-50: #e8dfec;
+    --text-black-900: #302e4d;
+    --text-black-700: #504e70;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    /* 当用户手动导航或者 CSSOM scrolling API 触发滚动操作时 */
+    scroll-behavior: smooth;
+}
+body{
+    background: var(--bg-black-900);
+}
+
+
+.hidden{
+    display: none;
+}
+
+.section .section-title{
+    color: var(--text-black-900);
+    font-size: 30px ;
+}
+
+
+/* 定义动画 */
+@keyframes carousel {
+
+    0%,
+    17.5% {
+        transform: translateZ(-35vw) rotateY(0);
+        /* 变换：沿z轴平移-35vw，绕y轴旋转0度 */
+    }
+
+    27.5%,
+    45% {
+        transform: translateZ(-35vw) rotateY(-120deg);
+        /* 变换：沿z轴平移-35vw，绕y轴旋转-120度 */
+    }
+
+    55%,
+    72.5% {
+        transform: translateZ(-35vw) rotateY(-240deg);
+        /* 变换：沿z轴平移-35vw，绕y轴旋转-240度 */
+    }
+
+    82.5%,
+    100% {
+        transform: translateZ(-35vw) rotateY(-360deg);
+        /* 变换：沿z轴平移-35vw，绕y轴旋转-360度 */
+    }
+}
+
+.section-sub-title{
+    padding-left: 30px;
+    color: var(--text-black-700);
+    font-size: 30px;
+}
+
+.nav {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: var(--body-color);
+    transition: all 0.3s ease;
+    z-index: 10000;
+}
+
+
+.nav .shell {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 250px;
+    padding: 10px 14px;
+    background: var(--shell-color);
+    transition: all 0.3s ease;
+    z-index: 100;
+}
+
+.nav .close {
+    width: 88px;
+}
+
+.nav .shell li {
```

Comment:
This CSS styling convention doesn't match what we're doing with CSS Modules.
This limits the CSS by nesting the classes, but with CSS modules you want names like 
.nav-shell-li or .nav-icon.

Our CSS classes are local and not global so we don't need to nest them.
Without CSS Modules, the way they name it here is also just bad. Long term they will run into maintenance issues with their global styles.

You likely should not copy this from where you got it and start more from scratch.